prql-compiler/src/lib.rs

```diff
@@ -65,7 +65,7 @@
 //! - ... compile, format & debug PRQL from command line.
 //!
 //!     ```sh
-//!     $ cargo install ripgrep
+//!     $ cargo install prql-compiler --features=cli
```

Comment:
WDYT about having this be a default feature? Atm installing from crates.io breaks.

I think I'll do a PR and then we can decide